### PR TITLE
Fix even more bad imports (#5711)

### LINKED ISSUE #5711: Divider custom example does not load in AMD
### Bug Report 
 
- __Package version(s)__: 6.41.0
- __Browser and OS versions__: 
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) No
 
Requested priority: (Blocking, High, Normal, Low) Normal
 
Products/sites affected: (if applicable)
 
#### Describe the issue:
 
The Divider custom example does not load when loaded as an AMD module with require.js.

The Divider import in VerticalDivider.Custom.Example should be:
`import { VerticalDivider, IVerticalDividerClassNames } from 'office-ui-fabric-react/lib/Divider';`
 
#### Actual behavior:
 
Uncaught JavaScript error thrown by require.js:

- `Uncaught Error: Script error for "office-ui-fabric-react/lib/components/Divider", needed by: office-ui-fabric-react/lib/components/Divider/examples/VerticalDivider.Custom.Example`
 
#### Expected behavior:
 
The example should load and render correctly.
 
### If applicable, please provide a repro:

````html 
<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <title></title>
    <link rel="stylesheet" href="https://unpkg.com/office-ui-fabric-react@6.41.0/dist/css/fabric.min.css">
    <script src="https://unpkg.com/babel-polyfill/dist/polyfill.min.js"></script>
    <script src="https://unpkg.com/babel-standalone/babel.min.js"></script>
</head>
<body>
    <div id="app"></div>
    <script src="https://unpkg.com/requirejs/require.js"></script>
    <script type="text/babel" data-presets="es2015,react">
        require.config({
            packages: [
                { name: '@microsoft/load-themed-styles', location: 'https://unpkg.com/@microsoft/load-themed-styles/lib-amd', main: 'index.js' },
                { name: '@uifabric/icons', location: 'https://unpkg.com/@uifabric/icons/lib-amd', main: 'index.js' },
                { name: '@uifabric/merge-styles', location: 'https://unpkg.com/@uifabric/merge-styles/lib-amd', main: 'index.js' },
                { name: '@uifabric/styling', location: 'https://unpkg.com/@uifabric/styling/lib-amd', main: 'index.js' },
                { name: '@uifabric/utilities', location: 'https://unpkg.com/@uifabric/utilities/lib-amd', main: 'index.js' }
            ],
            paths: {
                'office-ui-fabric-react/lib': 'https://unpkg.com/office-ui-fabric-react@6.41.0/lib-amd',
                'prop-types': 'https://unpkg.com/prop-types/prop-types.min',
                'react': 'https://unpkg.com/react/umd/react.production.min',
                'react-dom': 'https://unpkg.com/react-dom/umd/react-dom.production.min',
                'tslib': 'https://unpkg.com/tslib/tslib'
            }
        })([
            'react',
            'react-dom',
            'office-ui-fabric-react/lib/components/Divider/examples/VerticalDivider.Custom.Example',
            'office-ui-fabric-react/lib/Fabric',
            'office-ui-fabric-react/lib/Icons'
        ], (
            React,
            { render },
            { VerticalDividerCustomExample: DividerCustomExample },
            { Fabric },
            { initializeIcons }
        ) => {
            initializeIcons();

            function App() {
                return (
                    <Fabric>
                        <DividerCustomExample />
                    </Fabric>
                );
            }

            render(<App />, document.getElementById('app'));
        });
    </script>
</body>
</html>
````

### PR DESCRIPTION
**Pull request by cliffkoh**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #5711 
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Fix bad imports and it is now caught automatically.

### CODE REVIEW

**Comment by cliffkoh** [comment]
I know what the problem is - I check the real lib for imports from lib and so lint-imports now need to run after ts task..

File: scripts/tasks/lint-imports.js [hunk 1]
```diff
@@ -1,12 +1,42 @@
...
+    const potentialRushPath = path.resolve(curDir, 'rush.json');
+    if (fs.existsSync(potentialRushPath)) {
+      return potentialRushPath;
+    }
+    const parentFolder = path.dirname(curDir);
+    if (parentFolder === curDir) return undefined;
+    return findRushJson(parentFolder);
+  }
+
+  function getCurrentRushPkgName(rushJsonPath) {}
```
Comment (JasonGore) [inline scripts/tasks/lint-imports.js]: Is this dead code?

File: scripts/tasks/lint-imports.js [hunk 2]
```diff
@@ -97,20 +123,37 @@ module.exports = function(options) {
...
   function _evaluateImport(filePath, importPath, importErrors) {
+    let fullImportPath;
     if (importPath.indexOf('.') === 0) {
       // import is a file path. is this a file?
-      const fullImportPath = _evaluateImportPath(path.dirname(filePath), importPath);
+      fullImportPath = _evaluateImportPath(path.dirname(filePath), importPath);
+    } else {
+      const pkgNameMatch = importPath.match(pkgNameRegex);
+      if (pkgNameMatch === null) {
+        return;
```
Comment (JasonGore) [inline scripts/tasks/lint-imports.js]: What does this mean? There was no matching package found?

Comment (cliffkoh) [reply scripts/tasks/lint-imports.js]: Added a comment in the code that explains what it means.

**Comment by cliffkoh** [comment]
Ping @dzearing

**Pull request merged**